src/Downloader.ts

```diff
@@ -635,7 +665,9 @@ class Downloader {
         // Check for the etag and upload
         const etag = this.removeEtagWeakPrefix(mwResp.headers.etag);
         if (etag) {
-          this.s3.uploadBlob(stripHttpFromUrl(url), mwResp.data, etag);
+          this.s3.uploadBlob(stripHttpFromUrl(url), mwResp.data, etag,
+            !this.cssDependenceUrls.hasOwnProperty(mwResp.config.url) && this.webp ? 'webp' : '1'
```

Comment:
Same here, `!this.cssDependenceUrls.hasOwnProperty(mwResp.config.url)` shoudl be removed. Everything should be written with the same tag.